packages/auth/src/authClients/magicLink.ts

```diff
@@ -8,6 +8,8 @@ export interface AuthClientMagicLink extends AuthClient {
 }
 
 export const magicLink = (client: MagicLink): AuthClientMagicLink => {
+  let token: string | null
+  let expireTime: number
```

Comment:
I would set the default to now, then you can remove the extra check on line 24.
```suggestion
  let expireTime = Date.now(): number
```

Reply:
Good point ! 😄